Sources/Jobs/JobsProvider.swift

```diff
@@ -4,8 +4,25 @@ import NIO
 
 /// A provider used to setup the `Jobs` package
 public struct JobsProvider: Provider {
+    /// The number of seconds to wait before checking for the next job. Defaults to `1`
+    public var refreshInterval: TimeAmount
+    
+    /// The key that stores the data about a job. Defaults to `vapor_jobs`
+    public var persistenceKey: String
+    
+    /// The key to use for calling the command. Defaults to `jobs`
+    public var commandKey: String
+    
     /// Initializes the `Jobs` package
-    public init() { }
+    public init(
+        refreshInterval: TimeAmount = .seconds(1),
```

Comment:
I think this could be potentially confusing on both JobsConfiguration and the provider. Any reason not to have it on just the JobsConfiguration?

Reply:
Good point, will fix